README.md

```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://travis-ci.org/phac-nml/staramr.svg?branch=development)](https://travis-ci.org/phac-nml/staramr)
+[![Build Status](https://github.com/phac-nml/staramr/workflows/Integration%20Tests/badge.svg?branch=master&event=schedule)](https://github.com/phac-nml/staramr/actions?query=branch%master)
```

Comment:
Can you remove the TravisCI build status badge?

Reply:
Oops, fixed now.